xdsl/dialects/riscv.py

```diff
@@ -1676,6 +1676,7 @@ class MulOp(RdRsRsOperation):
     name = "riscv.mul"
 
 
+@irdl_op_definition
```

Comment:
Was this meant in this PR?

Reply:
Yes and no? It triggers errors as soon as I used IsolatedFromAbove, because it ended up checking those for traits, which was not a field because they were not properly IRDLed :slightly_smiling_face: 
So I'd say it's a fix indirectly triggered by adding this? :thinking: